### PR TITLE
Changelogs

### PR DESCRIPTION
**Pull request by nckturner**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Changelogs for recent releases: v1.20.0, v1.21.0, v1.22.1, v1.23.1

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nckturner** [comment]
/hold
Wait until failing tests on 1.21, 1.20 are fixed before publishing updated versions in the table.

File: docs/README.md [hunk 1]
```diff
@@ -47,10 +47,10 @@ The AWS cloud provider is released with a specific semantic version that correla
...
 |-----------------------------|-----------------------------------------------|
-| v1.24                       | v1.24.0-alpha.0                               |
-| v1.23                       | v1.23.0                                       |
-| v1.22                       | v1.22.0                                       |
-| v1.21                       | v1.21.0-alpha.0                               |
+| v1.24                       | v1.24.0                                       |
+| v1.23                       | v1.23.1                                       |
+| v1.22                       | v1.22.1                                       |
+| v1.21                       | v1.21.0                                       |
 | v1.20                       | v1.20.0-alpha.0                               |
```
Comment (hakman) [inline docs/README.md:54]:
```suggestion
| v1.20                       | v1.20.0                                       |
```

**Comment by hakman** [comment]
@nckturner I think all releases are unblocked now and ready to be published.

File: docs/README.md [hunk 2]
```diff
@@ -47,11 +47,11 @@ The AWS cloud provider is released with a specific semantic version that correla
...
 | Kubernetes Version          | Latest AWS Cloud Provider Release Version     |
 |-----------------------------|-----------------------------------------------|
-| v1.24                       | v1.24.0-alpha.0                               |
-| v1.23                       | v1.23.0                                       |
-| v1.22                       | v1.22.0                                       |
-| v1.21                       | v1.21.0-alpha.0                               |
-| v1.20                       | v1.20.0-alpha.0                               |
+| v1.24                       | v1.24.0                                       |
+| v1.23                       | v1.23.1                                       |
+| v1.22                       | v1.22.1                                       |
```
Comment (hakman) [inline docs/README.md:52]: Do we need a `v1.22.2` also? I don't see `v1.22.1` tagged in http://gcr.io/k8s-staging-provider-aws/cloud-controller-manager
```suggestion
| v1.22                       | v1.22.2                                       |
```

Comment (nckturner) [reply docs/README.md:52]: You're right... :confused:

Comment (nckturner) [reply docs/README.md:52]: For completeness this appears to be what we have: 

| Prod tag | Staging tag | Staging sha |
|-|-|-|
| v1.24.0 | v1.24.0 | sha256:9b1f6a6dd5de51cc30381187e0a3e236c6cebd34ca975074db3d4a3b0b2ca57c |
| v1.23.1 | v1.23.1 | sha256:96d591732ec106ba6b51595c10d66a68a141e4ef970af4e3d26bd9b042207ef3 |
| v1.22.1 | v20220513-v1.22.1 | sha256:6bccf3047f081889313180d4548f7d28fc4e5c6efa84e0dc08887500a2ee507d |
| v1.21.1 | v1.21.1 | sha256:30359d1236c610bdc139953cbc7b6ff66198ce9db1ddc131570c5e439b069581 |
| v1.21.0 | - | - |
| v1.20.1 | v1.20.1  | sha256:a5bfb84c5b6e6b6358f4b27b1029e7aafcece120cc2e9f34789a777f1f23d24b |
| v1.20.0 | - | - |

So we need a 1.22.2, but the last 1.22 build seems to have done nothing (https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/cloud-provider-aws-push-images/1526857358146277376), not sure what's going on there.

Comment (nckturner) [reply docs/README.md:52]: https://github.com/kubernetes/k8s.io/pull/3769

Comment (hakman) [reply docs/README.md:52]: I think we are good, I see the 1.22.2 tag in the staging repo.
I think this is the correct job for the tag https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/cloud-provider-aws-push-images/1529394025990197248.

**Comment by olemarkus** [comment]
Did I just see 1.22.3 released? Should this be updated as well?

**Review by olemarkus — APPROVED** [review]

**Review by hakman — APPROVED** [review]
👍

**Pull request merged**